sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.h

```diff
@@ -26,6 +26,10 @@
 #include "firmwares/rotorcraft/stabilization.h"
 #include "firmwares/rotorcraft/stabilization/stabilization_attitude_common_int.h"
 #include "firmwares/rotorcraft/stabilization/stabilization_attitude_ref_quat_int.h"
+#include "math/ActiveSetCtlAlloc/src/common/solveActiveSet.h"
+#ifdef USE_CHIBIOS_RTOS
+#include <ch.h>
```

Comment:
I would like to avoid including OS specific code here, are you sure it is really needed ?